actors/builtin/init/init_actor_state.go

```diff
@@ -55,7 +57,7 @@ func (s *State) ResolveAddress(store adt.Store, address addr.Address) (addr.Addr
 	}
 
 	// Not found.
-	return addr.Undef, errors.New("address not found")
+	return addr.Undef, ErrAddressNotFound
```

Comment:
If anything needs to actually inspect this I would much prefer we return a boolean indicating found or not.